packages/components/src/components/hds/form/key-value-inputs/index.ts

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import type { ComponentLike, WithBoundArgs } from '@glint/template';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { schedule } from '@ember/runloop';
+import { modifier } from 'ember-modifier';
+
+import {
+  ariaDescribedBy,
+  registerAriaDescriptionElement,
+  unregisterAriaDescriptionElement,
+} from '../../../../utils/hds-aria-described-by.ts';
+import { guidFor } from '@ember/object/internals';
+
+import HdsAlertComponent from '../../alert/index.ts';
+import HdsFormErrorComponent from '../error/index.ts';
+import HdsFormHelperTextComponent from '../helper-text/index.ts';
+import HdsFormKeyValueInputsDeleteRowButtonComponent from './delete-row-button.ts';
+import HdsFormKeyValueInputsFieldComponent from './field.ts';
+import HdsFormKeyValueInputsGenericComponent from './generic.ts';
+import HdsFormLegendComponent from '../legend/index.ts';
+
+import type { AriaDescribedByComponent } from '../../../../utils/hds-aria-described-by.ts';
+import type { HdsFormKeyValueInputsAddRowButtonSignature } from './add-row-button.ts';
+import type { HdsYieldSignature } from '../../yield/index.ts';
+
+const KEY_VALUE_PAIR_FIELD_SELECTOR = '.hds-form-key-value-inputs__field';
```

Comment:
```suggestion
const KEY_VALUE_INPUTS_FIELD_SELECTOR = '.hds-form-key-value-inputs__field';
```
[Nit] All the selectors in this section use the old name. They would need to be updated when used below as well.